src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -359,6 +361,10 @@ private class TaskLauncherActor(
       sender ! MatchedInstanceOps(offer.getId)
 
     case ActorOfferMatcher.MatchOffer(deadline, offer) =>
+      val active = instanceMap.values.filter(i => i.isActive).map(i => i.instanceId).mkString(",")
```

Comment:
if these are only for debugging, can they be in the debugging statement? or can there be an explicit debugging function that does this?